.github/workflows/github-actions-demo.yml

```diff
@@ -0,0 +1,26 @@
+name: leaderboard-backend CI
+
+on:
+  push:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+
+  Golang-CI:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checking out repository code
```

Comment:
Nit: Verbiage. Rather than saying `Checking out repository code`, something like `Check out repository code`, or even shorten it to `Checkout repo`. No need to be so verbose.

Reply:
Sounds good! Changed to `Checkout repo`